### ISSUE TITLE
Upgrading to v4.9.0 of the AzureRM provider results in errors.

### ISSUE THREAD

**Issue by DamaniN** [rubrikinc/terraform-azure-rubrik-cloud-cluster-elastic-storage#4]
After upgrading to v4.9.0 of the AzureRM provider the following errors occur:

```
╷
│ Error: Unsupported argument
│ 
│   on main.tf line 162, in resource "azurerm_network_interface" "cces_nic":
│  162:   enable_accelerated_networking = true
│ 
│ An argument named "enable_accelerated_networking" is not expected here.
╵
```

```
╷
│ Error: `subscription_id` is a required provider property when performing a plan/apply operation
│ 
│   with provider["registry.terraform.io/hashicorp/azurerm"],
│   on providers.tf line 18, in provider "azurerm":
│   18: provider "azurerm" {
│ 
╵
```

**Comment by DamaniN** [comment]
Fixed by [Updated to version 4.9.0 of the AzureRM provider.](https://github.com/rubrikinc/terraform-azure-rubrik-cloud-cluster-elastic-storage/commit/250c3be008a05ae329f038c1002a44788e362847) 250c3be008a05ae329f038c1002a44788e362847

**Comment by DamaniN** [comment]
Fixed by [Updated to version 4.9.0 of the AzureRM provider.](https://github.com/rubrikinc/terraform-azure-rubrik-cloud-cluster-elastic-storage/commit/250c3be008a05ae329f038c1002a44788e362847) 250c3be008a05ae329f038c1002a44788e362847
